Description: Add missing clean target to plugins to allow rebuild
Author: Kilian Krause <kilian@debian.org>

--- opal-3.10.1~dfsg.orig/samples/c_api/Makefile
+++ opal-3.10.1~dfsg/samples/c_api/Makefile
@@ -71,4 +71,4 @@ $(OBJDIR)/$(PROG): $(OBJECTS)
 depend:
 
 clean:
-	rm $(OBJECTS) $(OBJDIR)/$(PROG)
+	rm -f $(OBJECTS) $(OBJDIR)/$(PROG)
--- opal-3.10.1~dfsg.orig/samples/testpresent/Makefile
+++ opal-3.10.1~dfsg/samples/testpresent/Makefile
@@ -48,3 +48,5 @@ ifdef OPALDIR
 include $(OPALDIR)/opal_inc.mak
 endif
 
+clean:
+	rm -f $(OBJECTS) $(OBJDIR)/$(PROG)
--- opal-3.10.1~dfsg.orig/samples/opalmcu/Makefile
+++ opal-3.10.1~dfsg/samples/opalmcu/Makefile
@@ -48,3 +48,5 @@ ifdef OPALDIR
 include $(OPALDIR)/opal_inc.mak
 endif
 
+clean:
+	rm -f $(OBJECTS) $(OBJDIR)/$(PROG)
--- opal-3.10.1~dfsg.orig/samples/faxopal/Makefile
+++ opal-3.10.1~dfsg/samples/faxopal/Makefile
@@ -48,3 +48,5 @@ ifdef OPALDIR
 include $(OPALDIR)/opal_inc.mak
 endif
 
+clean:
+	rm -f $(OBJECTS) $(OBJDIR)/$(PROG)
--- opal-3.10.1~dfsg.orig/samples/opalecho/Makefile
+++ opal-3.10.1~dfsg/samples/opalecho/Makefile
@@ -44,3 +44,6 @@ endif
 ifdef OPALDIR
 include $(OPALDIR)/opal_inc.mak
 endif
+
+clean:
+	rm -f $(OBJECTS) $(OBJDIR)/$(PROG)
--- opal-3.10.1~dfsg.orig/samples/callgen/Makefile
+++ opal-3.10.1~dfsg/samples/callgen/Makefile
@@ -48,3 +48,5 @@ ifdef OPALDIR
 include $(OPALDIR)/opal_inc.mak
 endif
 
+clean:
+	rm -f $(OBJECTS) $(OBJDIR)/$(PROG)
--- opal-3.10.1~dfsg.orig/samples/ivropal/Makefile
+++ opal-3.10.1~dfsg/samples/ivropal/Makefile
@@ -48,3 +48,5 @@ ifdef OPALDIR
 include $(OPALDIR)/opal_inc.mak
 endif
 
+clean:
+	rm -f $(OBJECTS) $(OBJDIR)/$(PROG)
--- opal-3.10.1~dfsg.orig/samples/codectest/Makefile
+++ opal-3.10.1~dfsg/samples/codectest/Makefile
@@ -50,3 +50,5 @@ ifdef OPALDIR
 include $(OPALDIR)/opal_inc.mak
 endif
 
+clean:
+	rm -f $(OBJECTS) $(OBJDIR)/$(PROG)
